roles/openshift_manage_node/tasks/main.yml

```diff
@@ -1,22 +1,38 @@
+# Necessary because when you're on a node that's also a master the master will be
+# restarted after the node restarts docker and it will take up to 60 seconds for
+# systemd to start the master again
+- name: Wait for master API to become available before proceeding
+  # Using curl here since the uri module requires python-httplib2 and
+  # wait_for port doesn't provide health information.
+  command: >
+    curl --silent --cacert {{ openshift.common.config_base }}/master/ca.crt
+    {{ openshift_node_master_api_url }}/healthz/ready
```

Comment:
The fact is persisted for masters and relies on other master facts to build so this is passed in.